### PR TITLE
Add the "Language Support" Learn Page

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the "Language Support" Learn page.

> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md [hunk 1]
```diff
@@ -2,16 +2,127 @@
...
+  - /learn/tooling-guide/vs-code-extension/language-support/language-intelligence/
+  - /learn/tooling-guide/visual-studio-code-extension/language-support/language-intelligence
+  - /learn/tooling-guide/visual-studio-code-extension/language-support/language-intelligence/
   - /learn/tooling-guide/visual-studio-code-extension/language-support
 ---
 
-## Language Intelligence
+## IntelliSense
+
+The below are the features related to IntelliSense.
```
Comment (nadeeshaan) [inline learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md:30]: Do we need this line?

File: learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md [hunk 2]
```diff
@@ -2,16 +2,127 @@
...
+
+#### Signature Help
+
+When typing a function/method call expression, the signature help will show information such as the function/method call’s description and parameter information. Signature help will be triggered when typing the open parenthesis and comma.
+
+![Signature Help](/learn/images/signature-help.gif)
+
+## Code Navigation
+
+The below are the features related to code navigation.
```
Comment (nadeeshaan) [inline learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md:52]: Do we need this line?

File: learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md [hunk 3]
```diff
@@ -2,16 +2,127 @@
...
+
+#### Run and Debug Code Lenses
+
+Run and debug code lenses are shown for the entry points of the Ballerina project and for its test cases. The entry points include the main function and the services within the default module of the project.
+
+![Run and Debug Code Lenses](/learn/images/run-and-debug-code-lenses.gif)
+
+## Language Support Commands
+
+For more information, see (VSCode Commands)[/learn/tooling-guide/visual-studio-code-extension/vscode-commands/].
```
Comment (nadeeshaan) [inline learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md:118]: We can remove this part since it is not directly related to language support and will be covered with the extension capabilities.

**Review by nadeeshaan — CHANGES_REQUESTED** [review]

File: learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md (hunk 1 above)
Comment (praneesha) [reply learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md:30]: yes, better to have a description for the title.

File: learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md (hunk 2 above)
Comment (praneesha) [reply learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md:52]: yes, better to have a description for the title.

File: learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md (hunk 3 above)
Comment (praneesha) [reply learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md]: Removed.

File: learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md (hunk 1 above)
Comment (praneesha) [reply learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md:30]: Removed.

File: learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md (hunk 2 above)
Comment (praneesha) [reply learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md]: Removed.

**Review by nadeeshaan — CHANGES_REQUESTED** [review]

File: learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md [hunk 4]
```diff
@@ -2,16 +2,240 @@
...
+
+![Code Completion](/learn/images/code-completion.gif)
+
+#### Symbol Information on Hover
+
+When hovering over a symbol name, you will be provided with quick information about the particular symbol. For example, when hovering over a function name, you will be prompted with the associated documentation.
+
+![Symbol Information on Hover](/learn/images/symbol-information-on-hover.gif)
+
+#### Signature Help
```
[the remark this answers is not in the document]
Comment (praneesha) [reply learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md]: Fixed.

**Review by nadeeshaan — APPROVED** [review]
LGTM
